src/System.Runtime.Serialization.Json/tests/DataContractJsonSerializer.cs

```diff
@@ -2543,30 +2543,23 @@ public static void DCJS_VerifyDateTimeForFormatStringDCJsonSerSettings()
         dcjsSettings = new DataContractJsonSerializerSettings() { DateTimeFormat = jsonTypes.DTF_DMMMM };
         var actual3 = SerializeAndDeserialize(graph, "{\"DateTime\":\"1, mayo\",\"OffsetMinutes\":60}", dcjsSettings);
         Assert.NotNull(actual3);
-        Assert.True(actual3 == new DateTimeOffset(2017, 5, 1, 1, 0, 0, new TimeSpan(1, 0, 0)));
+        var expected3 = new DateTimeOffset(DateTime.Now.Year, 5, 1, 0, 0, 0, new TimeSpan(1, 0, 0));
+        Assert.True(actual3 == expected3, 
```

Comment:
`Assert.Equal` provides diffs and line number, which is good enough in most cases. But sometimes the line number may be inaccurate (shift 1 or 2 lines). As this particular test compares `DateTime` instances at different places, it's easier to find the failed assert by using custom message IMO. Yes, for this test, the variable name was helpful.